### ISSUE TITLE
Search results read-only

### ISSUE THREAD

**Issue by apiraino** [bastibe/org-journal#93]
Hi,

first of all, discovering this tool was to me like discovering fire, can't go back to a world without `org-journal`, thank you so much!

Now, when I search for a journal entry (`M-x org-journal-search`), results can only be opened in read-only. I don't understand whether there's a way to open them in write mode. Am I missing something or that's intentional (as in: doesn't make sense to rewrite old journal entries)?

Cheers

**Comment by bastibe** [comment]
Hi, and thank you for the praise.

View-mode is indeed intentional, mostly because it sets up `q` to close a view-mode buffer quickly. This allows for convenient journal browsing from the calendar, hitting `j` to view an entry, and `q` to exit back to the calendar. If you navigate to the previous/next entry from a view-mode entry, those entries will be opened in view-mode as well, so you can just `qqqqq` all entries that were opened this way.

In hindsight, I am not so sure if this was a good idea, and it has tripped me up a few times, too.

**Comment by apiraino** [comment]
Ok, that makes sense. Anyway I can workaround this behaviour with a `M-x toggle-read-only` to have it in write mode; should be ok, right?

Or alternatively, would it be useful a new shortcut [here](https://github.com/bastibe/org-journal/blob/ad61dcd1645de4292aef2e0450d41bee3b21fa4f/org-journal.el#L225), with something like:
```
 (define-key calendar-mode-map "e" 'org-journal-write-entry)
```
Not sure about what I am saying because too lazy to learn emacs lisp ;-)

**Comment by bastibe** [comment]
The correct workaround is `M-x view-mode`.

As for the shortcut, you can use `C-u i j` in calendar-mode, which will open the journal entry without view-mode, or `i j` to add a new header to the journal entry.

**Comment by apiraino** [comment]
Cool, thank you very much!

**Comment by bastibe** [comment]
Is this a satisfying solution to you? I'd be interested if you would prefer a different behavior as a customization option.

**Comment by apiraino** [comment]
I discovered that `e` is a shortcut for `M-x toggle-view` so I find the general interaction intuitive (for the emacs world standards).
In journal mode I open the calendar with `M-x calendar`, press `j` to open an item (or `C-c C-s` to search), then open the buffer I'm interested in view-mode; if I want to change it I only have to press `e`.

Look ok to me!

Unrelated, I have added an alias for `alias emacs_journal='emacs -f org-journal-mode'` because activating journal mode disables the modes for coding (I use [Prelude](https://github.com/bbatsov/prelude)), so I prefer editing my journal on another emacs instance.
